setup.py

```diff
@@ -14,68 +14,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""simpl packaging and installation."""
+"""simpl packaging, installation, and package attributes."""
 
-import ast
-import re
+import os
 from setuptools import find_packages
 from setuptools import setup
 
 
-DEPENDENCIES = [
+src_dir = os.path.dirname(os.path.realpath(__file__))
+
+about = {}
+with open(os.path.join(src_dir, 'simpl', '__about__.py')) as abt:
+    exec(abt.read(), about)
```

Comment:
@samstav - I think literal_eval compiles/accepts such a small set of operations that it is considered safe.

However, you are only executing a file which gets loaded by our code anyway and you're not doing any string manipulation or incorporating unknown inputs, so the risk is small. Merged.

If I were to suggest a risk, it would be that setup.py is usually run at a higher privilege (often root) than when called during runtime. So if __about__.py were manipulated in any way, it would be run as root. While that's a stretch, its obscure things like that that make me shy away from exec where possible. [this is coming from the person who put exec statements in checkmate, mind you]